common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CharacterProvider.java

```diff
@@ -71,10 +118,168 @@ public Optional<MapVisibility> getIconVisibility() {
+                ResourceLocation skin = SkinUtils.getSkin(user.getUuid());
+
+                if (!fullscreenMap) {
+                    // outline
+                    BufferedRenderUtils.drawRectBorders(
+                            poseStack,
+                            bufferSource,
+                            user.getRelationColor(),
+                            renderX,
+                            renderY,
+                            renderX + playerHeadRenderSize,
+                            renderY + playerHeadRenderSize,
+                            0,
+                            2);
+                }
+
+                // head
+                BufferedRenderUtils.drawTexturedRect(
+                        poseStack,
+                        bufferSource,
+                        skin,
+                        renderX,
+                        renderY,
+                        0,
+                        playerHeadRenderSize,
+                        playerHeadRenderSize,
+                        8,
+                        8,
+                        8,
+                        8,
+                        64,
+                        64);
+
+                // hat
+                BufferedRenderUtils.drawTexturedRect(
+                        poseStack,
+                        bufferSource,
+                        skin,
+                        renderX,
+                        renderY,
+                        1,
+                        playerHeadRenderSize,
+                        playerHeadRenderSize,
+                        40,
+                        8,
+                        8,
+                        8,
+                        64,
+                        64);
+
+                // health
+                if (fullscreenMap) {
+                    HealthTexture healthTexture = Managers.Feature.getFeatureInstance(MainMapFeature.class)
+                            .remotePlayerHealthTexture
+                            .get();
+                    BufferedRenderUtils.drawProgressBar(
+                            poseStack,
+                            bufferSource,
+                            Texture.HEALTH_BAR,
+                            renderX - 10,
+                            renderY + playerHeadRenderSize + 1,
+                            renderX + playerHeadRenderSize + 10,
+                            renderY + playerHeadRenderSize + 7,
+                            0,
+                            healthTexture.getTextureY1(),
+                            81,
+                            healthTexture.getTextureY2(),
+                            (float) user.getHealth().getProgress());
+                }
+
+                poseStack.popPose();
+            }
+        }
+    }
+
     private static final class CharacterLocation implements MapLocation {
         @Override
         public String getFeatureId() {
-            return "built-in:character:location";
+            return "player-you";
```

Comment:
Every other map feature used id's in a format like this, not the old "category-like" one.

Reply:
Hm? I thought I tried to use a hierarchical, colon-separated naming scheme for feature id:s?

Reply:
I don't think so. At least, you did not complain to me before that. I've not used hierarchical naming for feature ids, only categories. Check https://github.com/Wynntils/Static-Storage/blob/main/Reference/combat_locations_mapdata.json as an example of the format I use. As featureId is going to be something used in commands for example, I think my format makes sense. But you had this idea originally, so let me know.

Reply:
After thinking a bit about this, I think you are correct. My thinking was that the "fully qualified" name of a feature is it's category + feature id, so the feature id should be seen in this context.

That also means that the feature ids for hades players should probably be just the player name, not "player-", so we would get like `wynntils:player:friend:magicus`.

Reply:
Okay. I'll fix that up here, and the other PRs as well (in Static-Storage).

Reply:
Fixed (here).